app/src/main/java/com/asfoundation/wallet/ui/iab/PaymentMethodsInteract.kt

```diff
@@ -94,14 +93,8 @@ class PaymentMethodsInteract(private val walletService: WalletService,
 
   fun getLastUsedPaymentMethod(): String = inAppPurchaseInteractor.lastUsedPaymentMethod
 
-  fun getUserStatus(): Single<GamificationStats> {
-    return gamificationInteractor.getUserStats()
-  }
-
   fun hasAuthenticationPermission() = preferencesRepositoryType.hasAuthenticationPermission()
 
-  fun getLevels() = gamificationInteractor.getLevels()
-
   fun checkTransactionStateFromTransactionId(uid: String) =
```

Comment:
There is a link warning on this method. Since you'll rename the class name can you fix this as well?